packages/apps/clickhouse/values.yaml

```diff
@@ -42,3 +46,7 @@ backup:
   s3AccessKey: oobaiRus9pah8PhohL1ThaeTa4UVa7gu
   s3SecretKey: ju3eum4dekeich9ahM1te8waeGai0oog
   resticPassword: ChaXoveekoh6eigh4siesheeda2quai0
+
+image:
+  repository: clickhouse/clickhouse-server
+  tag: "24.2.2.71"
```

Comment:
We can't give users the oportunity to override image, because this will allow them to run any possible image in management cluster